README.md

```diff
@@ -1,57 +1,86 @@
-            [env: PORT=] [default: 4000]
 
-    -V, --version
-            Print version information
+## Running the Backend
+
+To run the Iroha 2 blockchain explorer backend, you have two options:
+
+**Option 1:**
+Execute the following command in your terminal:
+```bash
+./target/release/iroha2_explorer_web \
+  -c /path/to/client_config.json \
+  -p 8080  # may be env PORT, default is 4000
 ```
 
-Check:
+
+**Option 2:**
+Alternatively, you can use the `cargo run` command with additional options.
+This command will display the help information, including the available options and their descriptions.
+
 
 ```bash
-> curl http://localhost:4000
-Welcome to Iroha 2 Block Explorer!
+cargo run -- -h 
 ```
 
-## Deploy
+Here is a breakdown of the options you can use:
 
-- [Install Rust](https://www.rust-lang.org/tools/install)
-- Build binary:
+- `-c, --client-config <CLIENT_CONFIG>`: Specifies the path to the `iroha_client` JSON configuration file. The default path is set to `client_config.json` if not provided explicitly.
 
-  ```bash
-  cargo build --release
-  ```
+- `-h, --help`: Prints the help information, which provides an overview of the available options.
 
-- Prepare Iroha Client config ([reference](https://github.com/hyperledger/iroha/blob/ea45b5053018acd48340024800786ff5a3d0904d/docs/source/references/config.md)). **Define target peer location here**.
+- `-p, --port <PORT>`: Allows you to specify the port number on which the server will listen. You can set the port by providing the value after the flag, for example, `-p 8080`. If you don't provide this flag, the default 4000 port will be used. Additionally, you can set the `PORT` environment variable to specify the port.
 
-- Run web server:
+- `-V, --version`: Prints the version information of the Iroha 2 Explorer Backend.
 
-  ```bash
-  ./target/release/iroha2_explorer_web \
-      -c /path/to/client_config.json \
-      -p 8080 # may be env PORT, default is 4000
-  ```
+Feel free to adjust the command and options according to your specific setup and requirements.
+
+
+
+## Check the API
+
+Ensure that the explorer backend is functioning correctly by executing the following command:
+
+```bash
+> curl http://localhost:4000/api/v1
+Welcome to Iroha 2 Block Explorer!
+```
 
 ## API
 
 Refer to [Block Explorer API](api.md).
 
+
 ## Tools
 
-- [genesis-gen](./tools/genesis-gen/README.md) - genesis generator
\ No newline at end of file
+The following tools are available in conjunction with the Iroha 2 blockchain explorer:
+
+| Tool Name                        | Description                                                 |
+|----------------------------------|-------------------------------------------------------------|
+| [genesis-gen](./tools/genesis-gen/README.md) | Genesis generator (a tool to generate sample data).        |
+| [explorer-deploy-dev-tool](https://github.com/0x009922/explorer-deploy-dev-tool) | A tool for automating the deployment of Iroha and the explorer. |
```

Comment:
I would add something like "This tool might be out of date, but still be a reference of how to set up the project locally"